### PR TITLE
wix-ui-tpa | add types declarations

### PR DESCRIPTION
**Pull request by ayal**
add types declarations to package.json
should solve icons import

### CODE REVIEW

**Comment by jonathanadler** [comment]
Hey @ayal , thanks for the contribution!

In order for the build to run you'll need to sign your commits:
https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md

You can configure your commits to be signed, and then push a dummy commit in order for the build to run.

Thanks!!

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "sideEffects": false,
   "version": "2.12.0",
   "module": "./dist/es/src/index.js",
+  "types": "./dist/es/src/index.d.ts",
```
Comment (shlomitc) [inline package.json:7]:
```suggestion
  "types": "./dist/src/index.d.ts",
```
That's the way we do in other libraries

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_516/

**Comment by jonathanadler** [comment]
Closing this for now

**Pull request closed without merging**